src/main/java/io/cdap/plugin/dlp/Utils.java

```diff
@@ -32,6 +32,7 @@
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.stream.Collectors;
```

Comment:
why is this needed?

Reply:
We call `Collectors.toList()` on line 57